usr/include/include.mk

```diff
@@ -6,6 +6,6 @@ opencryptokiinclude_HEADERS =						\
 	usr/include/ec_curves.h usr/include/pqc_oids.h
 
 noinst_HEADERS +=							\
-	usr/include/apictl.h usr/include/local_types.h			\
-	usr/include/pkcs32.h usr/include/slotmgr.h usr/include/stdll.h	\
+	usr/include/apictl.h usr/include/local_types.h \
```

Comment:
why this change? I actually like to be the \ aligned, no ?

Reply:
Well, because the ones at the end of the previous block aren't aligned either. Instead of them being haphazard, I thought I'd clear them out.